src/Email/InvoiceEmailSender.php

```diff
@@ -6,49 +6,44 @@
 
 use Sylius\Component\Mailer\Sender\SenderInterface;
 use Sylius\InvoicingPlugin\Entity\InvoiceInterface;
-use Sylius\InvoicingPlugin\File\TemporaryFileSystemInterface;
+use Sylius\InvoicingPlugin\Filesystem\TemporaryFilesystem;
 use Sylius\InvoicingPlugin\Generator\InvoicePdfFileGeneratorInterface;
 
 final class InvoiceEmailSender implements InvoiceEmailSenderInterface
 {
     /** @var SenderInterface */
     private $emailSender;
 
-    /** @var TemporaryFileSystemInterface */
-    private $temporaryFilePathGenerator;
-
     /** @var InvoicePdfFileGeneratorInterface */
     private $invoicePdfFileGenerator;
 
+    /** @var TemporaryFilesystem */
+    private $temporaryFilesystem;
+
     public function __construct(
         SenderInterface $emailSender,
-        TemporaryFileSystemInterface $temporaryFilePathGenerator,
         InvoicePdfFileGeneratorInterface $invoicePdfFileGenerator
     ) {
         $this->emailSender = $emailSender;
-        $this->temporaryFilePathGenerator = $temporaryFilePathGenerator;
         $this->invoicePdfFileGenerator = $invoicePdfFileGenerator;
+        $this->temporaryFilesystem = new TemporaryFilesystem();
```

Comment:
There's no interface or DI for that as it's the very very internal workaround for attachments using Sylius Mailer. In case anyone wants to customise it, they should overwrite this class.